### PR TITLE
BaSP-TG-05: Timesheets table - Resources list

### PR DESCRIPTION
**Pull request by juliancit0**
In this PR I created React components to show a table with the list of resources from timesheets saved in the database.

For each row, you must be able to:
- Edit a timesheet.
- Delete a timesheet.

For the whole table I created a redirect button to 'forms' section that contains 'Add feature' to create a timesheet.

### CODE REVIEW

File: src/Components/TimeSheets/index.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/timesheets`);
+      const data = await response.json();
+      setTimesheets(data.data);
+    } catch (error) {
+      console.log(error);
+    }
+  }, []);
+
+  const deleteTimesheet = async (id) => {
+    if (confirm('Are you sure that you want to delete the task?')) {
```
Comment (TomasBettini) [inline src/Components/TimeSheets/index.jsx:26]: You must change the message "delete the task" to "delete the timesheet"

**Review by TomasBettini — DISMISSED** [review]
Good job!

File: src/Components/TimeSheets/index.jsx (hunk 1 above)
Comment (juliancit0) [reply src/Components/TimeSheets/index.jsx]: Hahahha, solved!

**Review by leimeter-joaquin — CHANGES_REQUESTED** [review]
When getting the list of timesheets I get this error on console:
`TypeError: Cannot read properties of null (reading 'name') at G (ListItem.jsx:14:25)`
Try to check for undefined or null values with ?. before using them

**Review by leimeter-joaquin — DISMISSED** [review]

**Review by leimeter-joaquin — APPROVED** [review]

**Review by Alfondalix — APPROVED** [review]

**Comment by disimoneleandro** [comment]
Approved for QA!

**Pull request merged**